code/datums/spells/lordspells.dm

```diff
@@ -0,0 +1,90 @@
+// Necrolord
+/obj/effect/proc_holder/spell/targeted/trigger/soulflare
+	name = "Soulflare"
+	desc = "Deals high damage to an enemy in 3 different damage types, as well as paralyzing them for 1 seconds If it hits an enemy in critical condition, it instantly kills them and lowers the cooldown permanently, to a maximum of 6."
+	school = "transmutation"
+	charge_max = 300
+	clothes_req = 1
+	invocation = "NEKROSIS"
+	invocation_type = "shout"
+	message = "<span class='notice'>Your head feels like it's being burned as you fall to the ground!</span>"
+	cooldown_min = 300
+	level_max = 0 // no upgrades because it allows you to get 0 cooldown if you wait with upgrades.
+
+	starting_spells = list("/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare")
+
+	action_icon_state = "soulflare"
+
+	var/soulflarecm = 300 // debug purposes
```

Comment:
it's there so I don't have to create a whole system of spaghetti code to see if soulflare's cooldown is 6 or below already.